src/components/Footer.jsx

```diff
@@ -0,0 +1,80 @@
+function Footer() {
```

Comment:
- [ ] OPTIONAL: I highly suggest refactoring this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Footer = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/57421823/f4d08976-18db-4c76-b484-495357b52e99)

I found 16.


That way, your code will meet the latest standards. 💪